### DISCUSSION TITLE
Can the Left Side Panel & the Notes Panels be Simultaneously Active in Mobile?

### DISCUSSION THREAD

**Discussion by ApplFn** [standardnotes/forum#2788] [Suggestions]
**Describe the issue**
Both the desktop and the web Standard Notes allow the left side panel (with views and folders) to remain always visible while simultaneously making the notes panels active and accessible. I can't find a way to do this in iOS or even iPadOS. You must pick one or the other. Is there currently a way to enable that feature?

**Smartphone (please complete the following information):**
 - Device: iPadAir3
 - OS: iPadOS 15.4 [same behavior in earlier versions]
 - 3.12.0 (114) iPadOS app

**Comment by stefan-yas** [comment]
Hey @ApplFn, it's not possible at the moment to have both the views and notes view panels open simultaneously on mobile, including tablets. It's a good suggestion though for larger displays, so it's something we may take into consideration as we improve UX/UI.

**Comment by ApplFn** [comment]
Thanks!
